shell/edit/monitoring.coreos.com.prometheusrule/AlertingRule.vue

```diff
@@ -524,4 +524,8 @@ export default {
   top: 0;
   right: 5px;
 }
+
+.promql-input {
+  width: 100%;
+}
```

Comment:
note: We can also consider applying `flex-grow: 1` to all children of labeled input to ensure that they consume the full width without requiring developers handle the edge cases with code mirror. Since this is a pattern that we only apply twice in Dashboard, I don't think that it will be worth the effort for this PR.